templates/registration/profile.html

```diff
@@ -3,17 +3,27 @@
 
 {% block content %}
     <h1>Profile</h1>
+    <h2 class="profile-header">Good morning {% firstof user.get_full_name user.get_username %}!</h2>
     {% if artists %}
+    <div class="investments">
```

Comment:
Oh, I just meant "a little bit" in the vague sense. It should be indented 4 spaces though- to show that this HTML will only be in the resulting file if `artists` is true.